R/bootstrap.nlsfit.R

```diff
@@ -211,44 +251,24 @@ print.bootstrapfit <- function(x, digits=2, ...) {
   summary.bootstrapfit(object=x, digits=digits, ...)
 }
 
-plot.bootstrapfit <- function(x, ..., xlim, ylim, rep=FALSE, col.line="black", col.band="gray", lty=c(1), lwd=c(1), xlab="x", ylab="y") {
+plot.bootstrapfit <- function(x, ..., xlim, ylim, rep=FALSE, col.line="black", col.band="gray", lty=c(1), lwd=c(1), xlab="x", ylab="y", samples=1000) {
```

Comment:
`samples` might be a bit misleading as a name here

Reply:
This is the gnuplot term. But I get your point. I can change it to whatever name you prefer.